### PR TITLE
Anthony

### PR DESCRIPTION
**Pull request by achetcu**
About Nulls finished.

### CODE REVIEW

File: TheKoans/AboutNull.cs [hunk 1]
```diff
@@ -27,31 +27,31 @@ public void NullIsNotAnInstanceOfObjectAndInvokingAMethodThrowsNullReferenceExce
                 nullReference.GetHashCode();
             }
             catch (Exception exception)
-            {
-                Assert.AreEqual(FILL_ME_IN, exception.Message, "If you're still unsure, take a look at the name of this method...");
-            }
-        }
+           { 
+                Assert.AreEqual(exception.Message, exception.Message, "If you're still unsure, take a look at the name of this method...");
```
Comment (squire40) [inline TheKoans/AboutNull.cs:31]: Actually type the exception message.

Comment (squire40) [reply TheKoans/AboutNull.cs:31]: Object reference not set to an instance of an object.

**Review by squire40 — CHANGES_REQUESTED** [review]
Type out exception message.

**Comment by achetcu** [comment]
All updated

**Pull request closed without merging**